### PR TITLE
Withdrawal_TestCase_PT_111 Update.

### PR DESCRIPTION
**Pull request by sahajisy**
```
#Author: Sahaj Balgunde
#Keywords Summary : Withdawal Test with Checkbox and Without Checkbox

@PT_111 @WithdrawAccount @Full @Smoke 
#Withdrawal PT_111
Feature: Withdraw user account

Scenario: User is clicking on Top button in Withdrawal Page.

Given Login To Platform Portal
Then Click Withdrawal link on Footer
Then Check Top button on Withdrawal Page

```

Updated 1 test case which was out of scope.

### CODE REVIEW

File: platform/src/test/java/platformfeatures/PF_Withdrawal_PT_111.feature [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+#Author: Sahaj Balgunde
+#Keywords Summary : Withdawal Test with Checkbox and Without Checkbox
+@WithdrawAccount @Full @Smoke @PT_111
```
Comment (csonparote) [inline platform/src/test/java/platformfeatures/PF_Withdrawal_PT_111.feature]: Putting this tag name @PT_111  here will execute all tags before it .
Plz rearrange tags as per desired execution

File: platform/src/test/java/platformfeatures/PF_Withdrawal_PT_111.feature [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+#Withdrawal PT_111
```
Comment (csonparote) [inline platform/src/test/java/platformfeatures/PF_Withdrawal_PT_111.feature]: Remove comments if not needed

**Review by csonparote — CHANGES_REQUESTED** [review]

File: platform/src/test/java/platformfeatures/PF_Withdrawal_PT_111.feature (hunk 2 above)
Comment (sahajisy) [reply platform/src/test/java/platformfeatures/PF_Withdrawal_PT_111.feature]: Done as requested.

File: platform/src/test/java/platformfeatures/PF_Withdrawal_PT_111.feature (hunk 1 above)
Comment (sahajisy) [reply platform/src/test/java/platformfeatures/PF_Withdrawal_PT_111.feature]: Done changes as requested.

**Review by csonparote — APPROVED** [review]
Plz import main into ur branch and test

**Review by csonparote — APPROVED** [review]
Plz import main into your branch and check if working

**Pull request merged**